dashboard/server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const axios = require('axios');
+require('dotenv').config()
+
+
+const app = express()
+app.get('/api/v1/endpoints', (req, res) => {
+    console.log('Mirror engaged: ', process.env.PBENCH_SERVER)
+    axios.get(
+            `${process.env.PBENCH_SERVER}/api/v1/endpoints`,
```

Comment:
Looks good,I have just one question
Where are we going to deploy our env variables?

Reply:
This is only for local dev mode, so there's nothing to "deploy" ... you've got your own `.env` in your local development tree, which takes effect while you're testing locally against a remote server. But when we deploy on a real server, we don't need the nodeJS reflector (or the `.env`) at all; the `index.html` "hack" here just loads the endpoints from `window.origin` by default, and all is good with the world.